cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1651,6 +1651,51 @@
                 }
               ]
             },
+            {
+              "Sid": "DynamoDBTableQuery",
```

Comment:
optional: if we want to start separating the policies that are only required by a HIT cluster you can use `ParallelClusterHITPolicies`